JS-files/dynamics.js

```diff
@@ -0,0 +1,84 @@
+// Data Base
+const trainer1 = {
+  name: 'M. S. Dhoni',
+  field: 'Indian Cricket Team Captain',
+  info: `Mahendra Singh Dhoni, (born July 7, 1981, Ranchi, India),
+     Indian cricketer whose rise to prominence in the early 21st century 
+     culminated in his captaincy of the Indian national team that won the 
+     one-day Cricket World Cup in 2011.`,
+  image: './trainers/MS Dhoni.jpeg',
+};
+
+const trainer2 = {
+  name: 'Sunil Chhetri',
+  field: 'Indian Football Team Captain',
+  info: `The all-time top scorer for the Indian football team and its 
+    captain for the better part of a decade, Sunil Chhetri is one of the 
+    most decorated footballers in India.`,
+  image: './trainers/Sunil_Chhetri_Indian_Football_Player.jpg',
+};
+
+const trainer3 = {
+  name: 'Mary Kom',
+  field: 'Gold medalist Indian boxer',
+  info: `Mangte Chungneijang Mary Kom OLY, famously known as Mary Kom, 
+    is a famous Indian boxer and an incumbent member of the Parliament of India. 
+    Mary Kom is an amateur boxer by profession and has also joined politics in 
+    recent times.`,
+  image: './trainers/mary-kom.jpeg',
+};
+
+const trainer4 = {
+  name: 'Vidyut Jammwal',
+  field: 'Bollywood Actor And Martial Art Expert',
+  info: `Vidyut Jammwal is an Indian film actor who predominantly works in Hindi 
+    films. He is also a trained martial artist, having learned Kalaripayattu (a martial 
+        art of Kerala) since the age of three. He is popularly known as "The New Age Action Hero."`,
+  image: './trainers/dc-Cover-vvm15gmruhtq6aktn304hc9j96-20180624233605.Medi.jpeg',
+};
+
+const trainer5 = {
+  name: 'Vikas Thakur',
+  field: 'Indian weightlifter',
+  info: `Vikas Thakur, an Indian weightlifter from Ludhiana (Punjab), was born on 
+    November 14, 1993. His hometown is in Patnoun, Himachal Pradesh, India. Weight of Vikas is 84 kg. 
+    His coach is B. S. Medhwan.`,
+  image: './trainers/vikas-thakur-asiad-afp.jpg',
+};
+
+const volunteers = {
+  name: 'Adarsh and Ankur',
+  field: 'Your Event Managers',
+  info: `they will guide you about all events ,their schedule , location,
+    requirements to participate, training fees and yeah of course your
+    event tickets. if you have any query you can reach out to them
+    at their cabin or desk`,
+  image: './trainers/volunteers.jpg',
+};
+
+const trainersDatabase = [trainer1, trainer2, trainer3, trainer4, trainer5, volunteers];
```

Comment:
- [ ] I would suggest that you don't declare each trainer in an independent variable, on large applications this could add extra performance issues. Instead, you could use the `trainersDatabase` array and define each JS object inside it, something like:

```js
const trainersDatabase = [
  {
    name: 'M. S. Dhoni',
    field: 'Indian Cricket Team Captain',
    info: `Mahendra Singh Dhoni, (born July 7, 1981, Ranchi, India),
       Indian cricketer whose rise to prominence in the early 21st century 
       culminated in his captaincy of the Indian national team that won the 
       one-day Cricket World Cup in 2011.`,
    image: './trainers/MS Dhoni.jpeg',
  },
  {
    name: 'M. S. Dhoni',
    field: 'Indian Cricket Team Captain',
    info: `Mahendra Singh Dhoni, (born July 7, 1981, Ranchi, India),
       Indian cricketer whose rise to prominence in the early 21st century 
       culminated in his captaincy of the Indian national team that won the 
       one-day Cricket World Cup in 2011.`,
    image: './trainers/MS Dhoni.jpeg',
  },
...
]

```

This will keep your logic simple and improve its readability! 🤓

Reply:
# thanks For this